analysis/src/SemanticTokens.ml

```diff
@@ -309,7 +309,14 @@ let command ~debug ~emitter ~path =
       Ast_iterator.default_iterator.expr iterator e
     | Pexp_record (cases, _) ->
       cases
-      |> List.iter (fun (label, _) -> emitter |> emitRecordLabel ~label ~debug);
+      |> List.filter_map (fun ((label : Longident.t Location.loc), _) ->
+             match label.txt with
+             | Longident.Lident s
+               when String.length s > 0
+                    && not (Char.equal s.[0] (Char.uppercase_ascii s.[0])) ->
```

Comment:
Could we make this a util (or do we already have one for it)? I can see this being useful elsewhere too.

Reply:
Done https://github.com/rescript-lang/rescript-vscode/pull/798/commits/7a24abdd9c202ed0e0f286844b60f47f2aa22f7c